modules/core/keeper/msg_server_test.go

```diff
@@ -217,171 +217,171 @@ func (suite *KeeperTestSuite) TestHandleRecvPacket() {
 }
 
 // tests the IBC handler receiving a packet using V2 protocol
-func (suite *KeeperTestSuite) TestRecvPacketV2() {
```

Comment:
I just commented out all these tests since i basically connected the testing package to channel/v2 packet handlers rather than packet-server.

I think a following PR should delete these tests along with packet-server package 🎉